apps/backend/src/app/modules/maps/maps.service.spec.ts

```diff
@@ -74,27 +80,27 @@ describe('MapsService', () => {
           ).rejects.toThrow(ForbiddenException);
         }
       }
-    })
+    });
 
-     it('should only non-logged in requests to access APPROVED and PUBLIC_TESTING maps', async () => {
-       for (const status of Enum.values(MapStatus)) {
-           db.mMap.findUnique.mockResolvedValueOnce({id: 1, status } as any);
+    it('should only non-logged in requests to access APPROVED and PUBLIC_TESTING maps', async () => {
+      for (const status of Enum.values(MapStatus)) {
+        db.mMap.findUnique.mockResolvedValueOnce({ id: 1, status } as any);
 
-           if ([MapStatus.APPROVED, MapStatus.PUBLIC_TESTING, MapStatus.FINAL_APPROVAL].includes(status)) {
-             expect(await service.getMapAndCheckReadAccess({ mapID: 1, })).toBeTruthy();
-           } else {
-             await expect(service.getMapAndCheckReadAccess({ mapID: 1 })).rejects.toThrow(ForbiddenException);
-           }
-       }
-     });
+        if ([MapStatus.APPROVED, MapStatus.PUBLIC_TESTING, MapStatus.FINAL_APPROVAL].includes(status)) {
+          expect(await service.getMapAndCheckReadAccess({ mapID: 1 })).toBeTruthy();
+        } else {
+          await expect(service.getMapAndCheckReadAccess({ mapID: 1 })).rejects.toThrow(ForbiddenException);
+        }
+      }
+    });
 
     // Map of what states should PASS access checks for each MapStatus,
     // otherwise should fail. Below code performs each check against each state.
     const tests = {
       [MapStatus.APPROVED]: 'any',
       [MapStatus.PUBLIC_TESTING]: 'any',
       [MapStatus.PRIVATE_TESTING]: ['admin', 'moderator', 'submitter', 'acceptedRequest', 'inCredits'],
-      [MapStatus.CONTENT_APPROVAL]: ['admin', 'moderator', 'submitter' ,'reviewer',  'acceptedRequest', 'inCredits'],
+      [MapStatus.CONTENT_APPROVAL]: ['admin', 'moderator', 'submitter', 'reviewer', 'acceptedRequest', 'inCredits'],
       [MapStatus.FINAL_APPROVAL]: 'any',
       [MapStatus.DISABLED]: ['admin', 'moderator']
     };
```

Comment:
Formatting again, same principle as previous comment, Prettier is good enough, doesn't help to have multiple autoformatters running at once.

Reply:
I believe prettier did take care of this!

Reply:
This formatting is wrong but there's a `prettier-ignore` comment above a parent block so shouldn't be touched. Looking at the code it looks unnecessary and I don't remind removing the comment, though little concerned if your editor/whatever CLI command you're using is ignoring this

Reply:
Gotcha, okay I'll be more weary of what it's up to. I thought I just had Prettier running, but in either case, I'll revert all these changes in this file and put it in a E2E :)

Reply:
Are you using a Prettier extension, just letting commit hooks format stuff, or something else? Curious as to what's going on here.